pyproject.toml

```diff
@@ -15,11 +15,39 @@ target-version = ["py36"]
 show_error_codes = true
 check_untyped_defs = true
 no_implicit_reexport = true
+warn_redundant_casts = true
 
 files = ["src/nacl"]
 
 [[tool.mypy.overrides]]
 module = [
     "nacl._sodium",
 ]
-ignore_missing_imports = true
\ No newline at end of file
+ignore_missing_imports = true
+
+[[tool.mypy.overrides]]
+module = [
+    "nacl.exceptions",
+]
+disallow_any_unimported = true
+disallow_any_expr = true
+disallow_any_decorated = true
+disallow_any_explicit = true
+disallow_any_generics = true
+disallow_subclassing_any = true
```

Comment:
Options which are probably too strict for most projects (so, if you are struggling later, these are the ones to remove):
```
disallow_any_unimported
disallow_any_expr
disallow_any_explicit
disallow_subclassing_any
```

If every dependency in nacl is typed (or stubs added), then the first and last options will probably be fine.